### DISCUSSION TITLE
RTMpose: couldn't get the same good quality skeleton as the online demo

### DISCUSSION THREAD

**Discussion by cqs0925** [open-mmlab/mmpose#2640] [General]
Hi all,
Thanks for the good work.
I successfully did the inference on a single cpu with minimal essential packages. However, the skeleton I extracted  is not as accurate as one from the website demo inference. (https://openxlab.org.cn/apps/detail/mmpose/RTMPose),  using the example image,(and I tried all the .onnx model from the official website https://platform.openmmlab.com/deploee)
Does the website demo uses different model? Will it be released?

Thanks again.

**Comment by Tau-J** [comment]
Hi @cqs0925 , thanks for using RTMPose. Updates to the models on the Deploee platform might lag behind the RTMPose official documentation. Therefore, you'll need to download the latest models from the RTMPose [homepage](https://github.com/open-mmlab/mmpose/tree/dev-1.x/projects/rtmpose). The model used in the online demo is trained on the `body8` datasets, and all models have already been released.

You can find the `.pth` body8 models as below:
![image](https://github.com/open-mmlab/mmpose/assets/13503330/74ab224d-05e8-4a6d-9067-55a215199087)

and you can convert them into `.onnx` via [deploee](https://platform.openmmlab.com/deploee/task-convert-list) online.

BTW, the code of online demo is also released, you can access to it [here](https://github.com/Tau-J/mmpose/blob/app-rtmpose/app.py)
